package.json

```diff
@@ -127,32 +134,27 @@
     "mobx-react": "^6.0.0",
     "mobx-state-tree": "3.14.1",
     "nock": "^13.2.1",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
     "prop-types": "^15.0.0",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0",
-    "react-is": "^16.13.1",
```

Comment:
we didn't appear to have to specify actual react versions here anymore...so removed